be/src/io/file_factory.cpp

```diff
@@ -147,8 +147,12 @@ Result<io::FileReaderSPtr> FileFactory::create_file_reader(
     switch (type) {
     case TFileType::FILE_LOCAL: {
         io::FileReaderSPtr file_reader;
-        RETURN_IF_ERROR_RESULT(io::global_local_filesystem()->open_file(
-                file_description.path, &file_reader, &reader_options));
+        std::string final_file = file_description.path;
+        if (final_file.rfind("file:", 0) == 0) { // Check if 'str' starts with 'prefix'
+            final_file = final_file.substr(5);
+        }
+        RETURN_IF_ERROR_RESULT(io::global_local_filesystem()->open_file(final_file, &file_reader,
+                                                                        &reader_options));
```

Comment:
file:: prefix is duplicated with type? Maybe we should parse when get type and check if they are same.